app/controllers/api/moves_filtered_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Api
+  class MovesFilteredController < MovesController
+  private
+
+    PERMITTED_CREATE_FILTER_PARAMS = [
```

Comment:
Same comment applies here with naming - ideally make these consistent across both new controllers, but definitely avoid using "create" in the name as that feels misleading.